service/src/main/java/bio/terra/tanagra/app/auth/AuthInterceptor.java

```diff
@@ -58,63 +61,50 @@ public boolean preHandle(HttpServletRequest request, HttpServletResponse respons
     }
 
     HandlerMethod method = (HandlerMethod) handler;
-    boolean isAuthRequired = false;
     ApiOperation apiOp = AnnotationUtils.findAnnotation(method.getMethod(), ApiOperation.class);
     if (apiOp != null) {
-      Authorization[] authorizations = apiOp.authorizations();
-      for (Authorization auth : apiOp.authorizations()) {
-        if (!auth.value().isEmpty()) {
-          LOGGER.info("Authorization required by endpoint: {}", request.getRequestURL().toString());
-          isAuthRequired = true;
-          break;
+      for (String tag : apiOp.tags()) {
+        if (!tag.isEmpty() && OPENAPI_TAG_AUTH_NOT_REQUIRED.equals(tag)) {
```

Comment:
`!tag.isEmpty()` is redundant since then it won't equal `OPENAPI_TAG_AUTH_NOT_REQUIRED` anyways?

Reply:
Good point. Removed.